app/components/windows/EditStreamInfo.vue.ts

```diff
@@ -144,10 +144,12 @@ export default class EditStreamInfo extends Vue {
         required: true,
         disabled: this.updatingInfo,
         description: this.isFacebook
+          /* eslint-disable */
```

Comment:
What doesn't eslint like about this?  Can't we just format it the way it wants?

Reply:
`ESlint` requires 12 spaces of indentation but `Eslint:prettier` plugin requires 14 spaces here.
That looks like conflicting rules that I can't fix in the code. This needs more investigation in the prettier config, but it seems like it's only one place in the code where we have this conflict.